src/dataflow/src/render/reduce.rs

```diff
@@ -834,95 +812,6 @@ where
         )
 }
 
-/// Builds a dataflow for hierarchical aggregation.
-///
-/// The dataflow repeatedly applies stages of reductions on progressively more coarse
-/// groupings, each of which refines the actual key grouping.
-fn build_hierarchical<G>(
-    collection: Collection<G, (Row, Row)>,
-    aggr: &AggregateFunc,
-    expected_group_size: Option<usize>,
-) -> Collection<G, (Row, Row)>
-where
-    G: Scope,
-    G::Timestamp: Lattice,
-{
-    let mut shifts = vec![];
-    let mut current = 4u64;
-
-    // We'll plan for an expected 4B records / key in the absense of hints.
-    // Note that here we will render what is essentially a 16-ary heap. At each reduce "layer",
```

Comment:
preserve this comment somewhere as its still relevant to the new version of build_hierarchical